study-builder/studies/pancan/emails/parental_consent_reminder.en.html

```diff
@@ -0,0 +1,13 @@
+#include ("emails.en")
+
+#set( $text = {
+    "salutation": $globalTranslationsFile.salutation,
```

Comment:
Can we use something like this (see one global translations file above) ?
And how can I check if it works ?

Reply:
Nice idea, I like it! We should definitely try to see if this works. I think the current way to check if the rendering works is to actually run it. Try invoking study-builder CLI with the `--create-emails --email-keys ...` flags (it'll actually create the emails in SendGrid, so you might want to go into there and clean things up afterwards).

I think we can easily add a new flag to _just_ try rendering the emails. I'll look into it.

Reply:
We agreed to step back from the idea for now.